python/ray/experimental/dag/test_actor_dag.py

```diff
@@ -0,0 +1,187 @@
+import pytest
```

Comment:
maybe I missed it, but I don't see a test where we pass a `ClassNode` into the constructor of another -- can you add that? Also one where it gets passed to a task.

Reply:
added and it passed as expected :)